packages/did-provider-ethr/src/__tests__/ethr-did-provider.test.ts

```diff
@@ -0,0 +1,189 @@
+import { IDIDManager, IKeyManager, MinimalImportableKey } from '@veramo/core-types'
+import { DIDManager, MemoryDIDStore } from '../../../did-manager/src'
+import { EthrDIDProvider } from '../ethr-did-provider'
+import { createAgent } from '../../../core/src'
+import { KeyManager, MemoryKeyStore, MemoryPrivateKeyStore } from '../../../key-manager/src'
+import { KeyManagementSystem } from '../../../kms-local/src'
+
+const PROVIDER = 'did:ethr:sepolia'
+const MOCK_DID = 'did:ethr:sepolia:0x76d331386cec35862a73aabdbfa5ef97cdac58cf'
+const KMS = 'local'
+const CONTROLLER_KEY = Object.freeze({
+  type: 'Secp256k1',
+  kid: 'controller-key',
+  publicKeyHex:
+    '3fdecd32358c5c3343f1096217baaf4ffffc68d64f5d82f3fb924e60191f6f3c0a46161f0f425d2c48ced2ec9a95f6af1993cdf0525250a4702407bce37f6269',
+  privateKeyHex: 'f090b28a3bd279049710908e913d1644df061a6316fe3e17792a18e5267c4bd5',
+  meta: {
+    algorithms: [
+      'ES256K',
+      'ES256K-R',
+      'eth_signTransaction',
+      'eth_signTypedData',
+      'eth_signMessage',
+      'eth_rawSign',
+    ],
+  },
+  kms: KMS,
+}) satisfies MinimalImportableKey
+
+const INFURA_API_KEY = ''
```

Comment:
// this key is safe to hardcode as it's limited to some did:ethr contract addresses
```suggestion
const INFURA_API_KEY = '3586660d179141e3801c3895de1c2eba'
```

Reply:
Thanks! I have added the key.